xs2a-adapter-aspsp-registry/src/main/java/de/adorsys/xs2a/adapter/registry/AspspCsvServiceImpl.java

```diff
@@ -56,6 +62,22 @@ public void importCsv(byte[] bytes) {
         aspspRepository.saveAll(aspsps);
     }
 
+    @Override
+    public void saveCsv() {
+        try {
+            if (CSV_ASPSP_ADAPTER_CONFIG_FILE_PATH == null || CSV_ASPSP_ADAPTER_CONFIG_FILE_PATH.isEmpty()) {
+//                String filePath = getClass().getClassLoader().getResource(DEFAULT_CSV_ASPSP_ADAPTER_CONFIG_FILE).getPath();
+//                Files.write(Paths.get(filePath), exportCsv());
+                throw new RegistryIOException("Adapter config file path is not set or Configuration File does not exist");
+            } else {
+                Files.write(Paths.get(CSV_ASPSP_ADAPTER_CONFIG_FILE_PATH), exportCsv());
+            }
+        } catch (IOException e) {
+            log.error("Exception occurred while re-writing aspsps into the CSV: {}", e.getMessage());
```

Comment:
Don't log and rethrow an exception. https://stackoverflow.com/questions/6639963/why-is-log-and-throw-considered-an-anti-pattern